proxy/proxy_test.go

```diff
@@ -1385,6 +1431,55 @@ var _ = Describe("Proxy", func() {
 				Expect(string(b)).To(MatchRegexp("gorouter_time:0"))
 			})
 
+			Context("in websocket requests", func() {
+				It("shows slowness in response_time, not gorouter_time", func() {
+					ln := test_util.RegisterWSHandler(r, "slow-ws-test", func(conn *websocket.Conn) {
+						msgBuf := make([]byte, 100)
+						n, err := conn.Read(msgBuf)
+						Expect(err).NotTo(HaveOccurred())
+						Expect(string(msgBuf[:n])).To(Equal("HELLO WEBSOCKET"))
+
+						time.Sleep(time.Second)
```

Comment:
Can you re-write this test to not require a sleep? We record the gorouter time with 6 decimal spots. I think it is accurate enough that gorouter_time and response_time will be different, even without a sleep. If there _must_ be a sleep due to flakiness (maybe your machine is very fast!) then can we have millisecond sleeps?